services/filesstore/s3store.go

```diff
@@ -142,16 +144,21 @@ func (b *S3FileBackend) TestConnection() error {
 	if exists {
 		mlog.Debug("Connection to S3 or minio is good. Bucket exists.")
```

Comment:
This is out of scope for this PR. But we should move these log lines from this package to the app layer. There is similar logging happening with the localstore also.

Reply:
I feel like backend-specific log messages should be placed close to the relevant code though?

Reply:
Logging is typically a concern of the higher level layer of an application. Low level packages should just be returning errors to indicate success or failures along with their context. That's why libraries rarely do logging on their own unless there are asynchronous tasks being performed somewhere else.

We don't need to do it in this PR. It was just a mental note.